### PR TITLE
WD-12299 Refreshed /solutions/secure-open-source

### PR DESCRIPTION
**Pull request by immortalcodes**
## Done

-  Refreshed /solutions/secure-open-source

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check if all is in accordance to [copydoc](https://docs.google.com/document/d/1mrvhkrVjCCJE39h96CPG_b_ZHWVN0GHzZ_6TdoudeYs/edit#heading=h.oy44zuppmsv2) and [figma design](https://www.figma.com/design/8JrAyKnWMU9FPJE0zJT2gu/Canonical.com%2Fsolutions%2Fsecure-open-source?node-id=0-1&t=GuOVZ4LegRxePrMe-0).
-  Please check demo [here](https://canonical-com-1293.demos.haus/solutions/secure-open-source)

## Issue / Card

Fixes # [WD-12327](https://warthogs.atlassian.net/browse/WD-12299)

[WD-12327]: https://warthogs.atlassian.net/browse/WD-12327?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/solutions/secure-open-source/index.html [hunk 1]
```diff
@@ -19,9 +19,9 @@
     <section class="p-suru--25-75">
       <div class="row--25-75">
         <div class="col">
-          <h1>Standardise your software supply chain with secure open source</h1>
+          <h1>Secure your open source software supply chain</h1>
```
Comment (samhotep) [inline templates/solutions/secure-open-source/index.html:22]: The formatting here seems to be slightly off; the title in the figma doc is not as heavy

Comment (samhotep) [reply templates/solutions/secure-open-source/index.html:22]: image
vs 
image

Comment (immortalcodes) [reply templates/solutions/secure-open-source/index.html:22]: The design mentions it to be H1, which is the case.
Also there are two design in the figma doc.

**Comment by Sophie-32** [comment]
There's still extra text in the last section that is not included in the copydoc

**Comment by lyubomir-popov** [comment]
please wrap this heading in a p-section--shallow:
image
- all hrs that don't span the full width should havce the class `is-muted`
- why icons? the figma file doesn't have them. They are different stroke thicknesses, and don't fit there, please remove them:
image
- the diagram is not readable, pls use a good resolution png:

image

the rest looks good!

**Comment by lyubomir-popov** [comment]
looks great thanks @immortalcodes!

**Review by samhotep — APPROVED** [review]
LGTM!

**Pull request merged**